config/routes.rb

```diff
@@ -52,6 +52,8 @@
     end
   end
 
+  resources :drug_stocks, only: [:create, :index]
```

Comment:
Same question as above re nesting to provide context and differentiate from the other drug stock controller.